bruno/main_collection/Delete File.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Delete File
+  type: http
+  seq: 5
+}
+
+delete {
+  url: {{baseUrl}}/api/files/{id}
```

Comment:
replace with `{{id}}`, like I did with the others, and set a variable for the id for this request. I guess you did not test this endpoint? 😉